framework/tlvf/tlvf.py

```diff
@@ -1085,13 +1094,34 @@ def addClassVarLenMethods(self, obj_meta, param_type, param_name, param_meta, pa
     # (list[]) which can only be the last member of each class, therefore no
     # action is needed for these lists.
     ##########################################################################
-    def addAllocationMarkers(self, obj_meta, func_name, param_meta, param_length, memmove):
+    def addAllocationMarkersCreate(self, obj_meta, func_name, param_meta, param_length, memmove):
```

Comment:
The Create and Alloc functions seem to be very similar. Isn't it possible to keep it as one function with some extra argument or something?

Reply:
I think it's better to leave them as 2 separated methods, since they dictate different flows:
'create' requires a call to 'add' method in order to finish the process, while 'alloc' doesn't.
I believe different methods make it easier to follow this scheme.

Reply:
Fine - so remove the func_name argument

Reply:
@LiorAmram can I remove it? This argument is func_name and it is being used inside the fucntion.
```
marker = "%s_%s_%s_%s" %(self.CODE_CLASS_ALLOC_INSERT, obj_meta.name, func_name, param_meta.name)
obj_meta.alloc_list.append((marker, func_name))
```

Reply:
You no longer need this argument, since each method has its own generating function.
So inside `addAllocationMarkersCreate`, you know that `func_name=create`.